inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/tool/excel/ExcelTool.java

```diff
@@ -0,0 +1,560 @@
+        }
+    }
+
+    /**
+     * Write the given objects to a list
+     * @param objects the objects to write
+     * @return a list of the objects
+     */
+    public static <E> List<Map<String, Object>> write2List(List<E> objects) {
+        Map<Field, String> fieldMap = new HashMap<>(objects.size());
+        Map<Field, ExcelCellDataTransfer> dataTransferEnumMap = new HashMap<>();
+        E e1 = objects.get(0);
+        Class<?> e1Class = e1.getClass();
+        Field[] fields = e1Class.getDeclaredFields();
+        if (fields.length == 0) {
+            return null;
+        } else {
+
+            for (Field field : fields) {
+                field.setAccessible(true);
+                ExcelField excel = field.getAnnotation(ExcelField.class);
+                if (excel != null) {
+                    String excelName = excel.name();
+                    fieldMap.put(field, excelName);
+                    ExcelCellDataTransfer excelCellDataTransfer = excel.x2oTransfer();
+                    dataTransferEnumMap.put(field, excelCellDataTransfer);
+                }
+            }
+
+            List<Map<String, Object>> list = new ArrayList<>();
+
+            for (E object : objects) {
+                Map<String, Object> map = new HashMap<>();
+                Class<?> objectClass = object.getClass();
+                Field[] declaredFields = objectClass.getDeclaredFields();
+                for (Field field : declaredFields) {
+                    field.setAccessible(true);
+                    String name = fieldMap.get(field);
+                    if (name != null) {
+                        Object value = null;
+
+                        try {
+                            value = field.get(object);
+                        } catch (IllegalAccessException e) {
+                            e.printStackTrace();
+                        }
+
+                        if (value != null) {
+                            ExcelCellDataTransfer excelCellDataTransfer =
+                                    dataTransferEnumMap.get(field);
+                            value = excelCellDataTransfer.parse2Text(value);
+                            map.put(name, value);
+                        }
+                    }
+                }
+
+                if (!map.isEmpty()) {
+                    list.add(map);
+                }
+            }
+
+            return list;
+        }
+    }
+
+    /**
+     * Read the given input stream and return a list of objects
+     * @param is the input stream to read
+     * @param eClass the class of the objects to return
+     * @return a list of the objects
+     * @throws IOException if there is an error reading the input stream
+     * @throws IllegalAccessException if there is an error accessing the class
+     * @throws InstantiationException if there is an error instantiating the class
+     */
+    public static <E extends ExcelImportTemplate> List<E> read(InputStream is, Class<E> eClass)
```

Comment:
This method is so long, could you please simply it by extracting some common methods?